app/service/rest.py

```diff
@@ -312,6 +318,29 @@ def update_whitelist(service_id):
         return '', 204
 
 
+@service_blueprint.route('/<uuid:service_id>/deactivate', methods=['POST'])
+def deactivate_service(service_id):
+    service = dao_fetch_service_by_id(service_id)
+
```

Comment:
okay that's fine, but we should probably look at everywhere else - and possibly remove the `@transactional` decorator, if we want to do this - virtually none of our other endpoints have db atomicity